openfl/experimental/workflow/interface/fl_spec.py

```diff
@@ -132,7 +123,110 @@ def runtime(self, runtime) -> None:
             raise TypeError(f"{runtime} is not a valid OpenFL Runtime")
         self._runtime = runtime
 
-    def _capture_instance_snapshot(self, kwargs):
+    def run(self) -> None:
+        """Starts the execution of the flow."""
+        # Submit flow to Runtime
+        if str(self._runtime) == "LocalRuntime":
+            self._run_local()
+        elif str(self._runtime) == "FederatedRuntime":
+            self._run_federated()
```

Comment:
it would be more elegant to have an Enum for the `runtime` attribute

Reply:
Thanks for the suggestion. As discussed in our call this will be taken up in a later release and has been added to the backlog